ripe/atlas/tools/probes/__init__.py

```diff
@@ -44,7 +44,7 @@ def get_many(cls, ids):
                 fetch_ids.append(str(pk))
 
         if fetch_ids:
-            kwargs = {"id__in": ",".join(fetch_ids)}
```

Comment:
hm, maybe. But, 1) I don't want to move away too much from KISS, 2) it has to be backwards compatible and that might add to complexity a bit.